pkg/sql/pgwire/pgerror/errors.go

```diff
@@ -151,14 +151,14 @@ func UnimplementedWithIssueError(issue int, msg string) error {
 // Unimplemented constructs an unimplemented feature error.
 //
 // `feature` is used for tracking, and is not included when the error printed.
-func Unimplemented(feature, msg string) *Error {
-	return UnimplementedWithDepth(1, feature, msg)
+func Unimplemented(feature, msg string, args ...interface{}) *Error {
+	return UnimplementedWithDepth(1, feature, msg, args...)
 }
 
 // UnimplementedWithDepth constructs an implemented feature error,
 // tracking the context at the specified depth.
-func UnimplementedWithDepth(depth int, feature, msg string) *Error {
-	err := NewErrorWithDepthf(depth+1, CodeFeatureNotSupportedError, "%s", msg)
+func UnimplementedWithDepth(depth int, feature, msg string, args ...interface{}) *Error {
+	err := NewErrorWithDepthf(depth+1, CodeFeatureNotSupportedError, msg, args...)
```

Comment:
This has the possibility to break stuff if `%` was used in any of the Unimplemented messages. Do we need to audit current use to make sure it's ok?

Reply:
it isn't -- there are a couple Sprintf'ers that could migrate to this, but I don't want to go refactoring right now.